src/admin/admin.routes.js

```diff
@@ -0,0 +1,70 @@
+(function () {
+    'use strict';
+
+    angular.module('motech-admin')
+        .config(adminRoutes);
+
+    adminRoutes.$inject = ['$stateProvider'];
+    function adminRoutes ($stateProvider) {
+        $stateProvider
+        .state('admin', {
+            url: '/admin',
+            ncyBreadcrumb: {
+                label: 'Admin'
+            }
+        })
+        .state('admin.bundles', {
+            url: '/bundles',
```

Comment:
It might make the most sense to leave the bundles implementation in their own file. I do think you made the correct change by prefixing each bundle route with 'admin.'

My issue is that the getBundle function at the bottom of the page makes the most sense within the directory with other bundles, or is an element that I should expose to the bundle controller directly.